infra/src/main/scala/ch/epfl/scala/services/storage/sql/SqlRepo.scala

```diff
@@ -97,3 +107,6 @@ class SqlRepo(conf: DbConf) extends DatabaseApi {
   private def run[A](v: doobie.ConnectionIO[A]): Future[A] =
     v.transact(xa).unsafeToFuture()
 }
+object SqlRepo {
+  val sizeOfInsertMany = 10000
```

Comment:
Does that mean that we might attempt to concurrently insert 10k entries?

Reply:
That means an insertMany request cannot be bigger than 10k entries. It is orthogonal to the size of the thread pool which control the level of concurrency.